### PR TITLE
Fix release notes that are failing `reno lint`

### PR DESCRIPTION
**Pull request by modernplumbing**
### What does this PR do?

A couple release notes were failing the linter. (Example: https://app.circleci.com/pipelines/github/DataDog/datadog-agent/53177/workflows/97ab7dbe-2d31-41ea-9b01-5fab17ed342a/jobs/702650)  This PR addresses those issues. 

These failing release notes were not a problem as of Aug 23 2022:
https://app.circleci.com/pipelines/github/DataDog/datadog-agent/52811/workflows/62f24746-55a7-457f-8578-c9d58b85525b/jobs/699186

or Aug 16 2022:
https://app.circleci.com/pipelines/github/DataDog/datadog-agent/52271/workflows/68edc1d3-3118-46fc-ab81-b5bcade7e083/jobs/694203

They're improperly formatted, but don't fail the job. It's unclear what changed, because the last changes to the .circleci folder were on Aug 5 2022. In any case, properly formatting these yaml files is a good thing.

### Reviewer's Checklist

- [x] If known, an appropriate milestone has been selected; otherwise the `Triage` milestone is set.
- [ ] Use the `major_change` label if your change either has a major impact on the code base, is impacting multiple teams or is changing important well-established internals of the Agent. This label will be use during QA to make sure each team pay extra attention to the changed behavior. For any customer facing change use a releasenote.
- [ ] A [release note](https://github.com/DataDog/datadog-agent/blob/main/docs/dev/contributing.md#reno) has been added or the `changelog/no-changelog` label has been applied.
- [ ] Changed code has automated tests for its functionality.
- [ ] Adequate QA/testing plan information is provided if the `qa/skip-qa` label is not applied.
- [x ] At least one `team/..` label has been applied, indicating the team(s) that should QA this change.
- [ x] If applicable, docs team has been notified or [an issue has been opened on the documentation repo](https://github.com/DataDog/documentation/issues/new).
- [ ] If applicable, the `need-change/operator` and `need-change/helm` labels have been applied.
- [ ] If applicable, the `k8s/<min-version>` label, indicating the lowest Kubernetes version compatible with this feature. 
- [ ] If applicable, the [config template](https://github.com/DataDog/datadog-agent/blob/main/pkg/config/config_template.yaml) has been updated.

### CODE REVIEW

**Review by maycmlee — COMMENTED** [review]
Nothing for docs to check

**Review by paulcacheux — APPROVED** [review]

**Comment by modernplumbing** [comment]
> Nothing for docs to check

would you mind approving, since Documentation is marked as a CODEOWNER on these files and this PR cannot be merged without Doc's approval

**Pull request merged**
